pkg/ssh/ssh.go

```diff
@@ -429,13 +430,36 @@ func RunSSHUpdateSubnet(host *models.Host, subnetName, importPath string) error
 
 // RunSSHSetupBuildEnv installs gcc, golang, rust and etc
 func RunSSHSetupBuildEnv(host *models.Host) error {
-	return RunOverSSH(
+	if err := RunOverSSH(
 		"Setup Build Env",
 		host,
 		constants.SSHScriptTimeout,
 		"shell/setupBuildEnv.sh",
-		scriptInputs{GoVersion: constants.BuildEnvGolangVersion},
-	)
+		scriptInputs{},
+	); err != nil {
+		return err
+	}
+	eg := &errgroup.Group{}
+	eg.SetLimit(2)
```

Comment:
I don't think you need to SetLimit for this?

Reply:
sure.